en/computer_vision/collision_prevention.md

```diff
@@ -4,7 +4,7 @@ _Collision Prevention_ may be used to automatically slow and stop a vehicle befo
 
 It can be enabled for multicopter vehicles in [Position mode](../flight_modes_mc/position.md), and can use sensor data from an offboard companion computer, offboard rangefinders over MAVLink, a rangefinder attached to the flight controller, or any combination of the above.
 
-Collision prevention may restrict vehicle maximum speed if the sensor range isn't large enough!
+Collision prevention may restrict vehicle maximum speed if the sensor range isn’t large enough!
```

Comment:
Generally I prefer the flat ' , so reverting this.

Reply:
yes, the change is from a vs code plugin, which i now removed. thanks for catching it.